src/dreyfus_index_updater.erl

```diff
@@ -31,6 +31,7 @@ update(IndexPid, Index) ->
     erlang:put(io_priority, {view_update, DbName, IndexName}),
     {ok, Db} = couch_db:open_int(DbName, []),
     try
+        {ok, ExcludeIdRevs} = purge_index(Db, IndexPid, Index),
```

Comment:
Hi @mayya-sharipova . If one doc is purged, there will be a purge seq and db update seq. The index delete/update will be run twice. So we collect the `ExcludeIdRevs` in `purge_index` to avoid this situation.
This change is in another PR(<https://github.com/cloudant-labs/dreyfus/pull/11>). 
The new PR I created is based on it and the commit history is kept.